site/content/tutorial/08-stores/01-writable-stores/app-a/App.svelte

```diff
@@ -4,14 +4,14 @@
 	import Decrementer from './Decrementer.svelte';
 	import Resetter from './Resetter.svelte';
 
-	let count_value;
+	let countValue;
 
-	count.subscribe(value => {
-		count_value = value;
+	const unsubscribe = count.subscribe(value => {
```

Comment:
revert this line because `unsubscribe` is unused
```suggestion
	count.subscribe(value => {
```

Reply:
done